### PR TITLE
Revert "Automatic deletion of bitwig.bitwig 4.0.7.108435"

### PR DESCRIPTION
**Pull request by ghost**
Reverts microsoft/winget-pkgs#51316

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/54223)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/b/bitwig/bitwig/4.0.7.108435/bitwig.bitwig.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+FileExtensions:
+- bwproject
+- bwtemplate
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads-eu.bitwig.com/stable/4.0.7/Bitwig%20Studio%204.0.7.msi
```
Comment (OfficialEsco) [inline manifests/b/bitwig/bitwig/4.0.7.108435/bitwig.bitwig.installer.yaml:19]:
```suggestion
  InstallerUrl: https://downloads.bitwig.com/stable/4.0.7/Bitwig%20Studio%204.0.7.msi
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/b/bitwig/bitwig/4.0.7.108435/bitwig.bitwig.installer.yaml (hunk 1 above)
Comment (Smithjuanita) [reply manifests/b/bitwig/bitwig/4.0.7.108435/bitwig.bitwig.installer.yaml]: Thanks so much for your help 🙏

**Pull request merged**